.github/workflows/chromatic.yml

```diff
@@ -0,0 +1,39 @@
+---
+name: Chromatic
+
+on: push
+
```

Comment:
Does this need to run on every change to jslib? If it just needs to run on changes to `./components`, can we add a `paths` block?

Reply:
We should be safe to only run it when components change.